postgraas_server/postgres_instance_driver.py

```diff
@@ -1,21 +1,20 @@
 __author__ = 'sebastian neubauer'
 import docker
-import hashlib
 import psycopg2
 import time
+import socket
 
 
-def get_unique_id(connection_dict):
-    return hashlib.sha1(str(frozenset(connection_dict.items()))).hexdigest()
+def get_hostname():
+    return socket.getfqdn()
```

Comment:
mmh, I thought about that one too, but it is not working in general I guess, quite often, the instance will be somehow rerouted, at least, the hostname FQDN must not be the hostname we pass to the user. So maybe remove this one here and maybe make another PR...